docs/keyboard-shortcuts.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Keyboard Shortcuts
+---
+
+export const Shortcut = ({ children }) => (
+  <kbd
+    style={{
+      margin: '0.2rem 0rem',
+    }}
+  >
+    {children}
+  </kbd>
+);
+
+A list of keyboard shortcuts that can be used within the Aerie UI.
+
+### Plan View
+
+**Timeline**
+
+The easiest way to zoom in and out is using <Shortcut>⌘</Shortcut> + <Shortcut>scroll</Shortcut> (MacOS) or <Shortcut>ctrl</Shortcut> + <Shortcut>scroll</Shortcut> (Windows) with the cursor over the timeline section. This works with a trackpad
```

Comment:
Might want to highlight both scroll to zoom and click and drag to pan while holding the modifier key

Reply:
good catch!